Binary files bfgminer-3.10.10.orig/bfgminer and bfgminer-3.10.10/bfgminer differ
Binary files bfgminer-3.10.10.orig/bfgminer-boinc_wrapper.o and bfgminer-3.10.10/bfgminer-boinc_wrapper.o differ
diff -urN -x result.txt -x stderr.txt -x configure -x config.status -x config.log -x aclocal.m4 -x Makefile -x Makefile.in bfgminer-3.10.10.orig/boinc_wrapper.cpp bfgminer-3.10.10/boinc_wrapper.cpp
--- bfgminer-3.10.10.orig/boinc_wrapper.cpp	2016-01-13 14:15:28.308311623 -0800
+++ bfgminer-3.10.10/boinc_wrapper.cpp	2016-01-13 14:23:31.011371280 -0800
@@ -3,6 +3,9 @@
 
 #include "boinc_api.h"
 #include <filesys.h>
+#include <string>
+
+using namespace std;
 
 BOINC_OPTIONS options;
 
@@ -58,3 +61,10 @@
 {
 	fclose(file);
 }
+
+void bw_boinc_get_result_physical_name(char *result_virtual_name, char *result_physical_name, int size)
+{
+	string resolved_name;
+	boinc_resolve_filename_s(result_virtual_name, resolved_name);
+	strncpy(result_physical_name,resolved_name.c_str(),256);
+}
diff -urN -x result.txt -x stderr.txt -x configure -x config.status -x config.log -x aclocal.m4 -x Makefile -x Makefile.in bfgminer-3.10.10.orig/boinc_wrapper.h bfgminer-3.10.10/boinc_wrapper.h
--- bfgminer-3.10.10.orig/boinc_wrapper.h	2016-01-13 14:15:28.308311623 -0800
+++ bfgminer-3.10.10/boinc_wrapper.h	2016-01-13 14:22:36.932238173 -0800
@@ -19,6 +19,7 @@
 void bw_boinc_show_message(int status, const char *msg, int is_notice);
 FILE* bw_boinc_fopen(char *path, char *mode);
 void bw_boinc_fclose(FILE *file);
+void bw_boinc_get_result_physical_name(char *result_virtual_name, char *result_physical_name, int size);
 
 #ifdef __cplusplus
 }
